pages/documentation/showcase.md

```diff
@@ -27,7 +27,6 @@ If your project is currently using USWDS and you would like to see it included i
 - [Agricultural Research Service - USDA](https://www.ars.usda.gov/)
 - [Scientific Discoveries - Agricultural Research Service - USDA](https://scientificdiscoveries.ars.usda.gov/)
 - [America’s Seed Fund - National Science Foundation](https://seedfund.nsf.gov/)
-- [Army Medical Department Center of History & Heritage](https://achh.army.mil)
```

Comment:
Site returns this error:
<img width="1258" alt="image" src="https://github.com/uswds/uswds-site/assets/3385219/3924071f-fe7b-49de-af04-ba404ca8b454">